css/style.css

```diff
@@ -1,34 +1,876 @@
+  letter-spacing: 0.03em;
+  background-color: rgb(231, 228, 228);
+  color: #6070ff;
+  padding: 0.3rem 0.3rem 0.3rem 0.3rem;
+  border-radius: 6px;
+}
+
+.but {
+  margin: 15px;
+  display: flex;
+  align-self: flex-start;
+  position: static;
+  height: 50px;
+  left: 12px;
+  top: 12px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  border: 1px solid blue;
+  border-radius: 16px;
+  padding: 0.7rem;
+}
+
+.lan,
+.frame,
+.skill {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 2rem 1rem 0;
+  margin: 0 1rem 1rem 1rem;
+}
+
+/*
+.framework {
+  border-bottom: solid;
+  border-bottom-color: #000;
+}
+
+.skills {
+  border-bottom: solid;
+  border-bottom-color: #000;
+
+} */
+
+.second-list-tyle {
+  margin-bottom: 0;
+}
+
+.first-list-style {
+  justify-content: space-between;
+}
+
+.but1 {
+  flex-basis: 100%;
+  padding-left: 10px;
+  padding-right: 10px;
+  margin-left: 0;
+  margin-bottom: 30px;
+}
+
+.pro {
+  display: flex;
+  align-items: center;
+  margin: 0 2.5rem 2rem 1rem;
+  background-color: #f7f7f9;
+  border-radius: 1rem;
+  padding: 0.5rem 0 0.5rem 0;
+  text-align: center;
+}
+
+.lang {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.jsi {
+  margin: 5px;
+}
+
+.js {
+  margin: 5px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 510;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+}
+
+.footer {
+  background-color: #6070ff;
+  font-style: normal;
+  font-family: Poppins, sans-serif;
+  height: 779px;
+  border-radius: 100px 0 0 0;
+  padding-top: 103px;
+  padding-right: 30px;
+  padding-left: 30px;
+}
+
+.contactme {
+  display: flex;
+  justify-content: center;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+.form {
+  flex-wrap: wrap;
+}
+
+.form-description {
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  margin-bottom: 60px;
+}
+
+.form-element {
+  font-family: Poppins, sans-serif;
   font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  flex: 1;
+  padding: 0 10px 0;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  width: 80%;
+}
+
+input.form-element {
+  height: 48px;
+  width: 327px;
+}
+
+textarea.form-element {
+  height: 170px;
+  flex: 1 0 auto;
+}
+
+.form-element::placeholder {
+  color: #b3bac5;
+}
+
+.flex-auto {
+  flex: 1 0 auto;
+}
+
+.but:enabled {
+  background-color: #fff;
+  border: 1px solid #7f8cff;
+}
+
+.but:hover {
+  background-color: #6070ff;
+}
```

Comment:
- [OPTIONAL] Great job implementing the button interactions (enable, hover, etc.). 💯 However, I suggest to change the text color to white `color: #fff` when you hover on the button 👍

Reply:
Thank you. Done!